### PR TITLE
Make health and sensor reports optional in battery service

### PR DESCRIPTION
**Pull request by Markushaldorsen**

### CODE REVIEW

File: adapter/service/battery/service.go [hunk 1]
```diff
@@ -49,12 +49,22 @@ type Reporter interface {
 	BatteryLevelReport() (level int64, state string, err error)
 	// BatteryAlarmReport returns a current battery alarm state.
 	BatteryAlarmReport() (AlarmReport, error)
+	// BatteryFullReport returns a current battery state.
+	BatteryFullReport() (FullReport, error)
+}
+
+type HealthReporter interface {
```
Comment (boguslaw-wojcik) [inline adapter/service/battery/service.go:56]: Add comment.

File: adapter/service/battery/service.go [hunk 2]
```diff
@@ -49,12 +49,22 @@ type Reporter interface {
...
+	Reporter
+
 	// BatteryHealthReport returns a current battery health state.
 	BatteryHealthReport() (int64, error)
+}
+
+type SensorReporter interface {
```
Comment (boguslaw-wojcik) [inline adapter/service/battery/service.go:63]: Add comment.

**Review by boguslaw-wojcik — DISMISSED** [review]

File: adapter/service/battery/service.go (hunk 1 above)
Comment (Markushaldorsen) [reply adapter/service/battery/service.go:58]: Done

File: adapter/service/battery/service.go (hunk 2 above)
Comment (Markushaldorsen) [reply adapter/service/battery/service.go:67]: Done

**Review by boguslaw-wojcik — APPROVED** [review]

**Pull request merged**